assets/guideline.css

```diff
@@ -0,0 +1,80 @@
+@media (max-width: 575.98px) {
+  .dropdown-toggle {
+    width: 100%;
+    white-space: normal;
+    word-break: break-word;
+  }
+}
+
+.dropdown-toggle {
+  transition: 0.15s;
+  background-color: var(--bs-secondary);
+}
+
+.dropdown-toggle:hover {
+  background-color: #ecf5c2;
+}
+
+.dropdown-menu {
+  box-shadow: rgba(0, 0, 0, 0.16) 0px 1px 4px;
+}
+
+.slider {
+  overflow: hidden;
+}
```

Comment:
I've actually tried using bs for this but for some reason it doesn't work, same goes for some other classes like rounded- and font-size, do you think its a bs problem? Hence for now, I think I should proceed with css instead of bs, wdyt?

Reply:
> I've actually tried using bs for this but for some reason it doesn't work, same goes for some other classes like rounded- and font-size, do you think its a bs problem? Hence for now, I think I should proceed with css instead of bs, wdyt?

@stingynoodles Ooooh, ik why....

See this:
```html
{{ $options := (dict "targetPath" "app.css" "includePaths" (slice "node_modules")) -}}
{{ $css := resources.Get "app.scss" | toCSS $options | postCSS | minify -}}
<link rel="stylesheet" href="{{ $css.Permalink }}"/>
```
I used `minify`, all unused bs classes are removed from `app.scss`. Since `rounded-*` (except `rounded-0`) are not used throughout the whole website (all pages), it is not included in the file.

Soooo, rerun your local server will do, i mean, rerun `npm run serve`.

A good issue, I should include it in the documentation.

Reply:
ohh i see alr thank you